pkg/rewardsUtils/rewardsUtils.go

```diff
@@ -3,36 +3,45 @@ package rewardsUtils
 import (
 	"bytes"
 	"fmt"
+	"text/template"
+
 	"github.com/Layr-Labs/sidecar/pkg/postgres/helpers"
 	"github.com/Layr-Labs/sidecar/pkg/utils"
 	"go.uber.org/zap"
 	"gorm.io/gorm"
-	"text/template"
 )
 
 var (
-	Table_1_ActiveRewards         = "gold_1_active_rewards"
-	Table_2_StakerRewardAmounts   = "gold_2_staker_reward_amounts"
-	Table_3_OperatorRewardAmounts = "gold_3_operator_reward_amounts"
-	Table_4_RewardsForAll         = "gold_4_rewards_for_all"
-	Table_5_RfaeStakers           = "gold_5_rfae_stakers"
-	Table_6_RfaeOperators         = "gold_6_rfae_operators"
-	Table_7_GoldStaging           = "gold_7_staging"
-	Table_8_GoldTable             = "gold_table"
+	Table_1_ActiveRewards           = "gold_1_active_rewards"
+	Table_2_StakerRewardAmounts     = "gold_2_staker_reward_amounts"
+	Table_3_OperatorRewardAmounts   = "gold_3_operator_reward_amounts"
+	Table_4_RewardsForAll           = "gold_4_rewards_for_all"
+	Table_5_RfaeStakers             = "gold_5_rfae_stakers"
+	Table_6_RfaeOperators           = "gold_6_rfae_operators"
+	Table_7_ActiveODRewards         = "gold_7_active_od_rewards"
+	Table_8_OperatorODRewardAmounts = "gold_8_operator_od_reward_amounts"
+	Table_9_StakerODRewardAmounts   = "gold_9_staker_od_reward_amounts"
+	Table_10_AvsODRewardAmounts     = "gold_10_avs_od_reward_amounts"
+	Table_7_GoldStaging             = "gold_7_staging"
+	Table_8_GoldTable               = "gold_table"
```

Comment:
Should be `Table_12_GoldTable`

Reply:
Done in [3d27474](https://github.com/Layr-Labs/sidecar/pull/131/commits/3d27474364a3d38c25d9cdf5ddc626fe872ac984)